ophyd/mca.py

```diff
@@ -96,7 +96,12 @@ def __init__(self, prefix, *, read_attrs=None,
         self.stage_sigs.update([(self.mode, 'PHA')])
 
     def stop(self):
-        self._stop.put(1)
+        self.stop_signal.put(1)
+
+
+class EpicsMCA(EpicsMCARecord):
```

Comment:
Bumping this because I keep forgetting to bring it up in person.

@dchabot any thoughts about the naming or logical break-up of these things?